components/admin_console/schema_admin_settings.jsx

```diff
@@ -870,6 +873,25 @@ export default class SchemaAdminSettings extends React.Component {
         this.setState({errorTooltip: isElipsis});
     }
 
+    submitSetting = async (callback, key, value) => {
```

Comment:
Is the `callback` parameter used somewhere?

Reply:
it's actually not, I just added it to make the interface similar to the `doSubmit` function that this is meant to mimic.
Based on what I can see `doSubmit = async (callback, getStateFromConfig)` also doesn't ever use the `callback` parameter. 

Think its worth it to remove it from both functions?

Reply:
I think it's worth it, yes. My philosophy is to add code if the usage is immediate or at least imminent.